crates/ruff_linter/src/rules/flake8_use_pathlib/rules/replaceable_by_pathlib.rs

```diff
@@ -156,7 +156,16 @@ pub(crate) fn replaceable_by_pathlib(checker: &Checker, call: &ExprCall) {
                 Some(OsReadlink.into())
             }
             // PTH208,
-            ["os", "listdir"] => Some(OsListdir.into()),
+            ["os", "listdir"] => {
+                if call
+                    .arguments
+                    .find_positional(0)
```

Comment:
I think you'd need to use `find_argument_value` or `find_argument` as `os.listdir` accepts the `path` argument as both positional and keyword.

Reply:
Thank you very much for your help! Indeed, I overlooked one scenario.